### DISCUSSION TITLE
Events:state for media_player node: Party mode (music) doesn't change status to next track

### DISCUSSION THREAD

**Discussion by cherjr** [zachowj/node-red-contrib-home-assistant-websocket#627] [Feature Request]
I'm using Events:state node for media_player. Everything's working fine, except status doesn't change to new track in Party mode (music) mode - status is changing only when it starts and when it ends (when I press stop) - [example](https://c2n.me/4fo50o4.png). It would be nice for it to change every time the next track starts playing.

**Comment by zachowj** [comment]
If you have `Ignore state change event when:  Current state equals previous state` checked in the event state node. It's only going to output when the state of the media player changes. So when it changes to `playing` and then again when it changes to `stop`. Unchecking that option will cause it to output when any value of the entity is updated. 

This will probably cause a lot more updates than you want but you can use a filter node to only continue if the track name has changed.
